### ISSUE TITLE
[CPT] I can trigger a BPMN timer event

### ISSUE THREAD

**Issue by saig0** [camunda/camunda#19176]
## Description

In Camunda Process Test, I want to trigger a BPMN timer event. Assuming that a timer event has a duration of 1 hour, I want to trigger the timer event immediately instead of waiting until the due date is reached.  

## Tasks
- Provide a test API to trigger a BPMN timer event  

requires #18997   
requires #18999 
related to #18815

**Comment by saig0** [comment]
## Solution idea

- Find the timer entity related to the given BPMN timer event (selected by process instance key and the element name/id)
- Modify the timer's due date to the current time
- As a result, the engine triggers the BPMN timer event

## Requirements

- New API to modify the due date: https://github.com/camunda/camunda/issues/18815
- New API to query timer entities

**Comment by marcellobarile** [comment]
To confirm, this task requires https://github.com/camunda/camunda/issues/18815, which isn't targeted for 8.10. Is this a mismatch, or are we not committed?

**Comment by saig0** [comment]
@marcellobarile you're right. Currently, this issue is not committed for 8.10, but it is a potential issue. We will revisit the target version during planning. If we decide on it, we will contribute the API.
